other-docs/guides/pre-release-modules.md

```diff
@@ -0,0 +1,25 @@
+# Updating Modules to Pre-Release Versions
+
+On occasion when there is a new feature ready for the next major version of Altis that is compatible with the previous version we may create a pre-release version.
```

Comment:
I think maybe we should call them "Preview Releases"; pre-release I think implies there will be an actual release?